test/libsolidity/syntaxTests/inheritance/override/triangle.sol

```diff
@@ -2,3 +2,5 @@ contract A { function f() public pure virtual {} }
 contract B is A { function f() public pure virtual override {} }
 contract C is A, B { }
 contract D is A, B { function f() public pure override(A, B) {} }
+// ----
+// TypeError: (116-138): Derived contract must override function "f". Function with the same name and parameter types defined in two or more base classes.
```

Comment:
The error is on contract C of course.

Reply:
Ah, ok, and ``contract C is B, A`` is invalid anyways... Then yes, I think this is exactly what we want - having ``A`` override again what ``B`` had overridden in it is weird anyways, so I think having to be explicit here is warranted.

By the way I think this is exactly the case we had in Zeppelin, but I could just let ``C`` not inherit from ``A`` there, since that resulted in the same overrides. But still it'd probably be good to have the ``=...`` workaround for this.

Reply:
By the way: in python's C3 the opposite is the case, so having ``A``, ``B is A`` and ``C is B, A`` is valid, but ``C is A, B`` is invalid... which actually makes more sense to be... so I'm again a bit confused about our C3 linearization :-)...

Reply:
Ah, ok, ours works in the opposite order, so the base mentioned last is the most derived base...

Reply:
Hm...

Reply:
Is there ever a case like this, in which I *couldn't* just remove ``A`` from the list without changing anything?

Reply:
The problem is, when you just add an empty contract between C and A it works:
```
contract A { function f() public pure virtual {} }
contract B is A { function f() public pure virtual override {} }
contract X is A {}
contract C is X, B { }
```